Code/Framework/AzToolsFramework/AzToolsFramework/API/EditorPythonConsoleBus.h

```diff
@@ -116,5 +116,12 @@ namespace AzToolsFramework
         virtual void OnExceptionMessage(AZStd::string_view message) = 0;
     };
     using EditorPythonConsoleNotificationBus = AZ::EBus<EditorPythonConsoleNotifications>;
+
+    // When using embedded Python, some platforms need to explicitly load the python library.
+    // For any modules that depend on 3rdParty::Python, this should be called.
+    namespace EmbeddedPython
+    {
+        void LoadLibPython();
+    }
```

Comment:
nit: Since this is highly important to be called before any modules use python I would put it at the top of the file, to be the first thing it's read.

Reply:
I had it at the top originally.  No problem, I can move it.